tests/network

```diff
@@ -150,24 +150,23 @@ networkTests
 lxc exec v1-macvlan -- apt-get install --no-install-recommends --yes ethtool
 lxc exec v1-macvlan -- ethtool -l enp5s0 | grep -c '^Combined:\s\+3$'  | grep -Fx 2
 
-# Check VM "agent.nic_config" works by reconfiguring eth0 to use parent and mtu settings.
+# Check VM "agent.nic_config" works by adding eth1 set to use parent name and MAC address.
 echo "=> Performing VM physical NIC agent.nic_config tests"
 lxc exec v1-physical -- "sync"
 lxc stop v1-physical -f
 lxc config set v1-physical agent.nic_config=true
-lxc config device set v1-physical eth0 nictype=physical parent="${parentNIC}v1" mtu=1400 name=eth0
+lxc config device add v1-physical eth1 nictype=physical parent="${parentNIC}v2" name=eth1
 lxc start v1-physical
 
 # Wait for lxd-agent to rename the interface.
 waitInstanceBooted v1-physical
 
 # Get the MAC address of the host NIC.
-parentNICMAC=$(cat /sys/class/net/"${parentNIC}v1"/address)
+parentNICMAC=$(cat /sys/class/net/"${parentNIC}v2"/address)
```

Comment:
This needs to be read before `v1-physical` is started as when it starts, the device and it's associated `/sys` files are gone from the host.

Reply:
I don't understand how it worked before:

```
$ cat /sys/class/net/enp2s0f0/address 
91:2a:c2:cc:a1:a2
$ lxc start v1-physical 
$ cat /sys/class/net/enp2s0f0/address
cat: /sys/class/net/enp2s0f0/address: No such file or directory
$ lxc delete -f v1-physical 
$ cat /sys/class/net/enp2s0f0/address
91:2a:c2:cc:a1:a2
```